build/optaplanner-build-parent/pom.xml

```diff
@@ -20,35 +20,11 @@
     <!-- Dependencies -->
     <!-- ************************************************************************ -->
 
-    <!-- SNAPSHOT and KIE dependency versions -->
-    <version.org.kie.kogito>2.0.0-SNAPSHOT</version.org.kie.kogito>
```

Comment:
Only "internal" dependencies stay in the `optaplanner-build-parent`; typically test dependencies or dependencies of `optaplanner-examples`. Every dependency (compile or runtime) of `optaplanner-quarkus-integration` moves to the new BOM, which becomes a BOM with 3rd party dependencies.